src/test/java/seedu/eduke8/storage/UserStorageTest.java

```diff
@@ -0,0 +1,98 @@
+package seedu.eduke8.storage;
+
+import org.json.simple.parser.ParseException;
+import org.junit.jupiter.api.Test;
+import seedu.eduke8.Eduke8Test;
+import seedu.eduke8.bookmark.BookmarkList;
+import seedu.eduke8.common.Displayable;
+import seedu.eduke8.exception.Eduke8Exception;
+import seedu.eduke8.question.Question;
+import seedu.eduke8.topic.Topic;
+import seedu.eduke8.topic.TopicList;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class UserStorageTest extends Eduke8Test {
```

Comment:
Looks like you have tried to cover as many possible scenarios as you can in these tests! Nice!